### DISCUSSION TITLE
hide timezone name in datetime field

### DISCUSSION THREAD

**Discussion by scramatte** [laravel/nova-issues#5916] [Ideas & Feature Requests]
Hello,

Can you add a feature to be able to hide timezone name displayed after datetime field

![image](https://github.com/laravel/nova-issues/assets/188766/3b4ffdb1-1754-49e9-8031-62239fdc278e)

Regards

**Comment by MarikaDiGirolamo** [comment]
I have the same issue

**Comment by fvaldes0109** [comment]
I have the same issue too
